src/empty-state-screen.tsx

```diff
@@ -1,3 +1,9 @@
+/*
+!!! IMPORTANT !!!
+Do not add new features here!
+This component is deprecated and will be removed
```

Comment:
if not used, I'd remove it now. Otherwise someone could decide to use it, even if deprecated.

Reply:
I deprecated it to follow a correct versioning and to remark the change in the changelog, but if it isn't important  I'll simply rename the component and make the new changes to it

Reply:
I think we should start thinking on prepare a new major version. We already have a bunch of deprecated stuff and I also have a branch with a breaking change which is difficult to avoid (https://github.com/Telefonica/mistica-web/pull/313#discussion_r700914561)